tests/unit_tests/test_wave_06.py

```diff
@@ -33,12 +33,10 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("Electronics")
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert item_a not in items
+    assert len(items) == 0
```

Comment:
It would be more explicit to check that `items` is an empty list like:
```suggestion
    assert items == []
```